domain/src/test/java/com/github/dodobest/domain/CalculatorTest.kt

```diff
@@ -0,0 +1,116 @@
+package com.github.dodobest.domain
+
+import org.junit.Assert.*
+import org.junit.Test
+
+class CalculatorTest {
+    private val calculator = Calculator()
+
+    @Test
+    fun test_plus_calculation() {
+        // when : 사용자가 덧셈 식을 입력하면
+        val actual: Double = calculator.evaluate("1+2+3")
+
+        // then : 올바른 덧셈 결과를 계산한다
+        assertEquals(6.0, actual, 0.00001)
+    }
+
+    @Test
+    fun test_minus_calculation() {
+        // when : 사용자가 뺄셈 식을 입력하면
+        val actual: Double = calculator.evaluate("10-2-3")
+
+        // then : 올바른 뺄셈 결과를 계산한다
+        assertEquals(5.0, actual, 0.00001)
+    }
+
+    @Test
+    fun test_multiple_calculation() {
+        // when : 사용자가 곱셈 식을 입력하면
+        val actual: Double = calculator.evaluate("10*-5*9")
+
+        // then : 올바른 곱셈 결과를 계산한다
+        assertEquals(-450.0, actual, 0.00001)
+    }
+
+    @Test
+    fun test_divide_calculation() {
+        // when : 사용자가 나눗셈 식을 입력하면
+        var actual: Double = calculator.evaluate("120/2/3")
+
+        // then : 올바른 나눗셈 결과를 계산한다
+        assertEquals(20.0, actual, 0.00001)
+
+        // when : 사용자가 나눗셈 식을 입력하면
+        actual = calculator.evaluate("10/3")
+
+        // then : 올바른 나눗셈 결과를 계산한다
+        assertEquals(3.3333333, actual, 0.00001)
+    }
+
+    @Test
+    fun test_null_input_calculation() {
+        // when : 사용자가 사칙연산 식에 null을 입력하면
+        val thrown: IllegalArgumentException = assertThrows(
+            IllegalArgumentException::class.java
+        ) { calculator.evaluate("120+null/3") }
+
+        // then : IllegalArgumentException 을 발생시킨다
+        assertEquals(IllegalArgumentException::class.java, thrown.javaClass)
```

Comment:
인스턴스 비교하는 assert 구문을 활용해보는 것은 어떨까요?
```suggestion
        assertThat(thrown).isInstanceOf(IllegalArgumentException::class.java)
```